addons/payment/views/payment_acquirer.xml

```diff
@@ -24,24 +24,31 @@
             <field name="arch" type="xml">
                 <form string="Payment Acquirer">
                     <sheet>
-                        <div class="oe_button_box" name="button_box">
+                        <field name="module_id" invisible="1"/>
+                        <field name="module_state" invisible="1"/>
+                        <div class="oe_button_box" name="button_box" attrs="{'invisible': ['&amp;', ('module_id', '!=', False), ('module_state', '!=', 'installed')]}">
                             <button class="oe_stat_button" name="%(payment_transaction_action_child)d"
                                 type="action" icon="fa-money" string="Payments">
                             </button>
                         </div>
-                        <field name="image" widget='image' class="oe_avatar oe_left" options='{"preview_image": "image_medium"}'/>
+                        <field name="image" widget='image' class="oe_avatar" options='{"preview_image": "image_medium"}'/>
                         <div class="oe_title">
                             <h1><field name="name" placeholder="Name"/></h1>
-                            <label for="provider" class="oe_edit_only"/>
-                            <field name="provider"/>
                         </div>
-                        <group name="acquirer_base" col="2">
+                        <div attrs="{'invisible': ['|', ('module_state', '=', 'installed'), ('module_id', '=', False)]}">
+                            <button type="object" class="btn btn-primary" name="button_immediate_install" string="Install"/>
+                            <div class="o_payment_acquirer_desc">
+                                <field name="description"/>
+                            </div>
+                        </div>
+                        <group name="acquirer_base" col="2" attrs="{'invisible': ['&amp;', ('module_id', '!=', False), ('module_state', '!=', 'installed')]}">
                             <group name="acquirer_left">
-                                <field name="environment" groups="base.group_no_one"/>
+                                <field name="provider" groups="base.group_no_one"/>
+                                <field name="environment"/>
```

Comment:
Was it intended to show the environement outside of Technical Settings ? I think it may be a leftover from the time you used the environment to check if a provider was installed or not.